website/content/docs/commands/auth-methods/update.mdx

```diff
@@ -251,6 +251,21 @@ The following are options are specific to OIDC auth-methods in addition to the c
 
 - `-max-age` `(string: "")` - Indicates the OIDC "max_age" parameter sent to the provider.
 
+- `prompts` `(string: "")` - Indicates whether the OIDC authorization server should display authentication or consent user interface prompts.
```

Comment:
Should we say: "display reauthentication, account selection or consent user prompts"? Just to highlight the `select_account` prompt